tests/core/DependencyBearTest.py

```diff
@@ -0,0 +1,217 @@
+from concurrent.futures import ThreadPoolExecutor
+
+from coalib.core.DependencyBear import DependencyBear
+from coalib.core.FileBear import FileBear
+from coalib.core.ProjectBear import ProjectBear
+from coalib.settings.Section import Section
+
+from tests.core.CoreTestBase import CoreTestBase
+
+
+class TestProjectBear(ProjectBear):
+
+    def analyze(self, files):
+        yield ', '.join('{}({})'.format(filename, len(files[filename]))
+                        for filename in sorted(files))
+
+
+class TestFileBear(FileBear):
+
+    def analyze(self, filename, file):
+        yield '{}:{}'.format(filename, len(file))
```

Comment:
The test just doesn't use raw files ;) I would code other test bears once I have to test support for that ;)